executor/write.go

```diff
@@ -193,6 +200,18 @@ func (e *DeleteExec) deleteMultiTables() error {
 	for _, t := range e.Tables {
 		tblMap[t.TableInfo.ID] = append(tblMap[t.TableInfo.ID], t)
 	}
+	var colPosInfos []tblColPosInfo
+	for id, cols := range e.SelectExec.Schema().TblID2Handle {
+		tbl := e.tblID2Table[id]
+		for _, col := range cols {
+			if names, ok := tblMap[id]; !ok || !isMatchTableName(names, col) {
```

Comment:
`isMatchTableName` is only used in this function, so this name is too general.
I think make it a method of `DeleteExec` and move rename to `isDeletingTable` is better.

```
if !e.isDeletingTable(tbleMap, col) {
    continue
}
```
Would be more clear.